includes/data-events/connectors/class-mailchimp.php

```diff
@@ -62,6 +62,20 @@ private static function get_audience_id() {
 		return ! empty( $audience_id ) ? $audience_id : false;
 	}
 
+	/**
+	 * Get default reader MailChimp status.
+	 *
+	 * @return string MailChimp status slug, 'transactional' or 'subscriber'. (Default: 'transactional').
+	 */
+	private static function get_default_reader_status() {
+		$allowed_statuses = [
+			'transactional',
+			'subscribed',
+		];
+		$default_status = Reader_Activation::get_setting( 'mailchimp_reader_default_status' );
+		return in_array( $default_status, $allowed_statuses ) ? $default_status : 'transactional';
```

Comment:
```suggestion
		return in_array( $default_status, $allowed_statuses, true ) ? $default_status : 'transactional';
```

CI might throw a warning if we don't use `in_array` with strict type checking